platform-includes/capture-error/go.mdx

```diff
@@ -1,10 +1,11 @@
-To capture an event in Go, you can pass any struct implementing an `error` interface to `CaptureException()`. If you use a 3rd party library instead of native `errors` package, we'll do our best to extract a stack trace.
+To capture an event in Go, you can pass any struct implementing an `error` interface to `CaptureException()`. If you use a 3rd party library instead of native `errors` package and it implements `Unwrap() error` or `Cause() error` method, we'll follow the chain of errors and extract the stack trace from the root cause.
```

Comment:
```suggestion
To capture an event in Go, you can pass any struct implementing an `error` interface to `CaptureException()`. If you use a 3rd party library instead of the native `errors` package and it implements the `Unwrap() error` or `Cause() error` method, we'll follow the chain of errors and extract the stack trace from the root cause.
```